pkg/simple/client/elasticsearch/esclient.go

```diff
@@ -137,25 +137,33 @@ type EmptyField struct {
 }
 
 type StatisticsAggs struct {
```

Comment:
I believe Agg means Aggregation? don't use initial here, isn't easy to understand

Reply:
since the `aggs` is the field name used by es search body. Should we keep it in line with es?
![image](https://user-images.githubusercontent.com/22350668/57666575-02915780-7633-11e9-8ad7-c32dd642f38a.png)

Reply:
`aggregations` has been used in response. So you can find type `StatisticsAggregations` for response in code to distinguish these two naming

![image](https://user-images.githubusercontent.com/22350668/57666754-a1b64f00-7633-11e9-8085-97ec829d06e2.png)

Reply:
OK, better to add comments to clarify